src/tokens/Assets.tsx

```diff
@@ -601,6 +602,9 @@ const styles = StyleSheet.create({
     color: Colors.gray3,
     textAlign: 'center',
   },
+  noNftsTextContainer: {
+    paddingHorizontal: Spacing.Regular16,
```

Comment:
This should this be 24 per the design [here](https://www.figma.com/file/OE6RFzqXAMqU9ygHtNYyof/Assets-Page-(Multichain)?node-id=2307%3A16037&mode=dev)